client/app/pages/matches/matches.component.html

```diff
@@ -1,66 +1,54 @@
-                        <th *ngFor="let missingTable of missingFieldsArray(amountofMissingHeaderFields())"></th>
-                        <th>Action</th>
-                    </tr>
-                    <tr *ngFor="let match of (matches() | teamMatchesPipe: filter)">
-                        <td>{{match.matchId}}</td>
-                        <td>{{match.stage | upperCaseFirstLetter}}</td>
-                        <td>{{match.startTime | date : 'shortTime'}}</td>
-                        <td>{{match.endTime | date : 'shortTime'}}</td>
-                        <td *ngFor="let matchTeam of match.matchTeams"
-                            [innerHTML]="matchTeam.teamNumber | boldPartNumber : filter">
-                        </td>
-                        <td *ngFor="let field of missingFieldsArray(amountOfMissingFields())"></td>
-                        <td class="actions">
-                            <div class="button" data-open="model-edit" (click)="setEditModel(match)">
-                                <i class="fa fa-edit"></i>
-                            </div>
-                            <div class="alert button" data-open="model-delete" (click)="setDeleteModel(match)">
-                                <i class="fa fa-trash-alt"></i>
-                            </div>
-                        </td>
-                    </tr>
+<div class="ui container">
+    <div class="ui padded grid" id="matches">
+        <div class="column">
+            <div class="ui segment">
+                <table class="ui single line very basic scrollable table">
+                    <thead class="center aligned">
+                        <tr>
+                            <th>Stage</th>
+                            <th>Start Time</th>
+                            <th>End Time</th>
+                            <th *ngFor="let table of tables()">
+                                {{table.tableName}}
+                            </th>
+                            <th *ngFor="let missingTable of missingFieldsArray(amountofMissingHeaderFields())"></th>
+                            <th>Action</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let match of (matches() | teamMatchesPipe: filter)">
+                            <td>{{match.stage | upperCaseFirstLetter}}</td>
+                            <td>{{match.startTime | date : 'shortTime'}}</td>
+                            <td>{{match.endTime | date : 'shortTime'}}</td>
+                            <td *ngFor="let matchTeam of match.matchTeams"
+                                [innerHTML]="matchTeam.teamNumber | boldPartNumber : filter">
+                            </td>
+                            <td *ngFor="let field of missingFieldsArray(amountOfMissingFields())"></td>
+                            <td class="center aligned">
+                                <i class="ui primary edit link icon" (click)="setEditModel(team)"></i>
```

Comment:
this need to set match to the edit modal

Reply:
Fixed